lib/restbase.js

```diff
@@ -164,6 +165,9 @@ RESTBase.prototype.defaultListingHandler = function(value, restbase, req) {
 
 // Special handling for external web requests
 RESTBase.prototype.defaultWebRequestHandler = function(req) {
+    // enforce the usage of UA
+    req.headers = req.headers || {};
+    req.headers['User-Agent'] = req.headers['User-Agent'] || this.rb_config.user_agent;
```

Comment:
> notice the number of the RFC as being identical to the RB port

No coincidence ;)

+1 for fixing in a follow-up. RFC 7230 states: "Each header field consists of a case-insensitive field name followed by a colon (":"), optional leading whitespace, the field value, and optional trailing whitespace."